test/blackbox-tests/test-cases/github4821.t/run.t

```diff
@@ -0,0 +1,15 @@
+  $ touch foo.opam
+  $ cat > dune-project<<EOF
+  > (lang dune 2.0)
+  > EOF
+  $ cat > dune <<EOF
+  > (library (name foo) (public_name foo) (enabled_if false))
+  > EOF
+  $ export DUNE_BUILD_DIR=$(mktemp -d -t github4821XXXXXX);
+  > dune build @install;
+  > cat $DUNE_BUILD_DIR/default/foo.install | sed s#$DUNE_BUILD_DIR#DUNE_BUILD_DIR#g;
```

Comment:
It might be preferable to print the dune-package file instead. Otherwise, it's not clear if the library is excluded because it is disabled or it is simply empty.